app-tasks/rf/src/rf/uploads/geotiff/create_footprints.py

```diff
@@ -93,6 +94,29 @@ def transform_polygon_coordinates(feature, src_crs, target_crs):
     return feature
 
 
+def add_crs_metadata(mask_tif_path):
+    """Adds CRS to tifs that lack it
```

Comment:
Good call, I've [added that](https://github.com/azavea/raster-foundry/pull/1932/files#diff-1745ee6d15e48808170bc7216d9b10f8R134).